.travis.yml

```diff
@@ -4,10 +4,8 @@ python:
   - "2.7"
 
```

Comment:
adding `sudo: false' tells Travis to use their container-based system, which I've found is quite a bit faster. Though now that we're removing support for all these environments, we won't even notice :)